packages/graphql/src/utils/get-token.ts

```diff
@@ -63,3 +62,11 @@ export function getToken(context: Context): string | undefined {
 
     return token;
 }
+
+export function parseBearerToken(bearerAuth: string): string {
+    const token = bearerAuth.split("Bearer ")[1];
+    if (!token) {
+        debug("Authorization header was not in expected format 'Bearer <token>'");
```

Comment:
**question:** surely this is just going to break down the line if we don't throw and error here?

Reply:
Possibly, this logic is actually the same as our current token logic for normal requests (same file, line 58).

I'm not sure what the fallback is for or its implications, but I think that falls under authentication improvements task we have pending to do